src/inimpkg/commands.nim

```diff
@@ -0,0 +1,29 @@
+import algorithm
+from strutils import join
+from os import getCurrentDir, getHomeDir, setCurrentDir, walkDir, absolutePath, lastPathPart
+from osproc import execCmd
+
+proc ls*(dir: string = getCurrentDir()): seq[string] =
+  ## Print contents of a directory
+  for kind, path in walkDir(dir):
+    result.add(lastPathPart(path))
+  result.sort()
+
+
+proc cd*(dir: string = getHomeDir()): string =
+  ## Change dir
+  try:
+    setCurrentDir(dir)
+  except OsError as e:
+    return e.msg
+  getCurrentDir()
+
+
+proc pwd*(): string =
+  ## Print current path
+  getCurrentDir()
+
```

Comment:
dude, I expected you to shell out to these commands, but they're pure nim! That's pure genius, and might even work on Windows! :laughing:

Reply:
Yeah, I think most of the os commands should be fairly compatible. Not sure what else to add, but it should be easy enough for anyone to add a new command.